charts/persistence/values.yaml

```diff
@@ -69,3 +69,8 @@ trino:
       hive.s3.path-style-access=true
       hive.s3.endpoint=http://minio:80
       hive.s3.ssl.enabled=false
+
+# minio operator
+operator:
+  enabled: false
```

Comment:
I think enabled false by default is the right call here 👍 
Let's keep it that way for the external charts as opt in.